src/ensembl/src/shared/components/badged-button/BadgedButton.test.tsx

```diff
@@ -26,66 +26,48 @@ const defaultProps = {
   badgeContent: faker.lorem.words()
 };
 
-const renderButton = (
-  ButtonComponent: React.FunctionComponent<any>,
-  props: any = defaultProps
-) => mount(<ButtonComponent {...props} />);
+const renderButton = (props: Partial<BadgedButtonProps> = {}) =>
+  render(
+    <BadgedButton {...defaultProps} {...props}>
+      <Button onClick={onClick}>{faker.lorem.words()}</Button>
+    </BadgedButton>
+  );
 
 describe('BadgedButton', () => {
-  let wrapper: any;
-
   beforeEach(() => {
-    wrapper = renderButton(BadgedButton, {
-      ...defaultProps,
-      children: <Button onClick={onClick}>{faker.lorem.words()}</Button>
-    });
-  });
-
-  afterEach(() => {
     jest.resetAllMocks();
   });
-
   it('renders the passed in button', () => {
-    expect(wrapper.find('button').length).toEqual(1);
+    const container = renderButton().container;
```

Comment:
Just for reference, this could just be written as `const {container} = renderButton()`

Reply:
I have made the changes and push it.